### ISSUE TITLE
Data collection: Restrict region to drive, select specific path

### ISSUE THREAD

**Issue by zuoxingdong** [ai-tor/DeepGTAV#33]
Because we want to have a training and testing dataset separately. Is it possible to restrict the car only drive in some sub-region, or even better manually select certain paths, because we do not want to touch the test set.

**Comment by ai-tor** [comment]
Hi @zuoxingdong ,

This is not currently implemented, what you can do is to drive for let's say, 8 hours to save a training dataset. And then stop and restart in a another random location, and save a dataset for 2 hours as testing set.
